src/seedu/addressbook/data/person/Address.java

```diff
@@ -52,6 +52,11 @@ public int hashCode() {
         return value.hashCode();
     }
 
+    @Override
+    public String getPrintableString(){
+    	return this.toString();
```

Comment:
i don't think your `toString()` is returning in the format specified?
hence it may not be apt to just return the to string from this method, no?